src/main/java/uk/gov/homeoffice/digital/sas/timecard/model/TimeEntry.java

```diff
@@ -0,0 +1,70 @@
+package uk.gov.homeoffice.digital.sas.timecard.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.Type;
+import uk.gov.homeoffice.digital.sas.jparest.annotation.Resource;
+import uk.gov.homeoffice.digital.sas.jparest.models.BaseEntity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+import java.util.UUID;
+
+@Resource(path = "time-entry")
+@Entity(name = "time_entry")
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@NoArgsConstructor
+@Getter
+@Setter
+public class TimeEntry extends BaseEntity {
+
+    private int version;
+
+    @NotNull
+    @Column(name = "owner_id")
```

Comment:
Just wondering about the column names here. If I remember correctly we resolved the issue with the spring naming strategy and we were able to use camelcase for the java entity variable names without the '@column' annotation and spring should do the conversion to snake case behind the scenes.

Reply:
This improves more readability and best practise i think Daleel :)